github/Team.py

```diff
@@ -234,13 +243,23 @@ def get_repo_permission(self, repo):
         except UnknownObjectException:
             return None
 
+    @deprecated(
```

Comment:
This is missing an import: from deprecated import deprecated at the top of a file.

Reply:
Thanks. Fixed in https://github.com/PyGithub/PyGithub/pull/1509/commits/0373fcde52f3b71da75e3eb277c52b265a315793